pkg/compute/kube/client_test.go

```diff
@@ -2315,8 +2328,9 @@ func TestK8sServicesToMeshServices(t *testing.T) {
 				Spec: corev1.ServiceSpec{
 					Ports: []corev1.ServicePort{
 						{
-							Name: "p1",
-							Port: 80,
+							Name:     "p1",
```

Comment:
nit: In the above comment for this test, we could clarify the default is only set to http if the port's protocol is TCP

Reply:
Done, thanks